jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -269,6 +270,21 @@ public Containerizer setToolName(String toolName) {
     return this;
   }
 
+  /**
+   * Sets the alwaysCacheBaseImage property to control the optimization which skips downloading base
```

Comment:
Sorry, missed this one. From the jib-core library perspective, there is no such property. Start the sentence with

"Controls the optimization..."

Reply:
Changed it.